js/src/components/PremiumBenefitsList.js

```diff
@@ -0,0 +1,45 @@
+/* global yoastPremiumBenefitsL10n */
+import React from "react";
+import styled from "styled-components";
+import interpolateComponents from "interpolate-components";
+
+let localizedData = null;
+if ( window.yoastPremiumBenefitsL10n ) {
+	localizedData = yoastPremiumBenefitsL10n;
+}
+
+const StyledList = styled.ul`
+	list-style: none;
+	margin: 0 0 16px;
+
+	li {
+		margin: 5px 0 0 0;
+		padding-left: 16px;
+	}
+
+	span[aria-hidden="true"]:before {
+		margin: 0 8px 0 -16px;
+		font-weight: bold;
+		content: "+";
+	}
+`;
+
+const PremiumBenefitsList = () => {
+	return (
+		localizedData &&
+			<StyledList role="list">
+				{ localizedData.intl.map( ( benefit, index ) => {
+					let formattedBenefit = benefit.replace( "<strong>", "{{strong}}" ).replace( "</strong>", "{{/strong}}" );
```

Comment:
What is the reason you put the extra step in here? You skipped this for the AddKeyword modal.

Reply:
Can I say "I don't remember?" 😄  removed.

Reply:
That's because the strings come from PHP, and they should be reusable in PHP in other places, see 
> Forgot to mention this PR also introduces a class to print out the list of Premium benefits, in the hope they can be standardized at some point. See #9120

so they must use `<strong>`, then in JS I need to do replace them to `{{strong}}` before passing them to interpolate components